### DISCUSSION TITLE
Suggested settings for training

### DISCUSSION THREAD

**Discussion by medledan** [TheLastBen/fast-stable-diffusion#1445] [Q&A]
I am trying to train a model for a single specific character (face and body) . I have 37 images of various facial features and expressions and full body images. What are the suggestions for:

I have not been able to understand much of this. 
UNet_Training_Steps
Text_Encoder_Training_Steps

I have no Concept Images because I wanted maximum variety. 
Also, how do you know if a model has been overtrained? 

Thanks for any help.

**Comment by TheLastBen** [comment]
keep the text encoder to default settings, reduce the number of images to 10, start with the default settings, then resume the training of the unet(only) for 200 steps at 8e-6 lr at a time until you get the desired results.
